stock-tracker/src/features/keyStats/component/keyStats.tsx

```diff
@@ -31,17 +35,17 @@ export const KeyStats = () => {
                 </td>
                 <td className="keystats__grid__display1__value">
                   {response && response.previousClose
-                    ? Math.abs(response.previousClose).toFixed(2)
+                    ? twoDecimalPlaces(response.previousClose)
```

Comment:
suggestion: Rather than compute this in the render every time, why not make a function that takes your object keys and applies `twoDecimalPlaces` to each key?

Reply:
That would be a nice little refactor, maybe we could try pair program this is another PR?